sormas-app/app/src/main/java/de/symeda/sormas/app/settings/SettingsFragment.java

```diff
@@ -88,12 +93,22 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
 		binding.resynchronizeData.setOnClickListener(v -> repullData());
 		binding.showSyncLog.setOnClickListener(v -> openSyncLog());
 		binding.logout.setOnClickListener(v -> logout());
+		binding.kexLbds.setOnClickListener(v -> kexLbds());
+		binding.syncPersonLbds.setOnClickListener(v -> LbdsIntentSender.sendPersonsLbds(getContext()));
+		binding.syncCaseLbds.setOnClickListener(v -> LbdsIntentSender.sendCasesLbds(getContext()));
```

Comment:
synCases should send the new cases including the related persons. There should not be a separate button for sending persons

Reply:
Has been discussed: for the prototype phase we will keep two buttons for better control. Will be consolidated after.